csrankings-r.csv

```diff
@@ -942,12 +942,12 @@ Robert Kleinberg,Cornell University,http://www.cs.cornell.edu/~rdk,zkvW8FQAAAAJ
 Robert Kooima,Louisiana State University,http://csc.lsu.edu/~kooima,OcIwjtEAAAAJ
 Robert Kraut,Carnegie Mellon University,http://kraut.hciresearch.org,HKGYvu4AAAAJ
 Robert Krauthgamer,Weizmann Institute of Science,https://www.weizmann.ac.il/pages/search/people?language=english&single=1&person_id=4211,M3yUD0cAAAAJ
-Robert L. (Scot) Drysdale III,Dartmouth College,http://www.cs.dartmouth.edu/~scot,NOSCHOLARPAGE
+Robert L. (Scot) Drysdale III,Dartmouth College,http://cs.dartmouth.edu/~scot,mn1QyYgAAAAJ

```

Comment:
Emeritus; please remove this and all corresponding entries. https://web.cs.dartmouth.edu/people

Reply:
Bump @quattrinili

Reply:
done